zap/src/output/luau/client.rs

```diff
@@ -550,11 +749,12 @@ impl<'src> ClientOutput<'src> {
 
 		self.push_callback_lists();
 
-		if self
-			.config
-			.evdecls
-			.iter()
-			.any(|ev| ev.evty == EvType::Reliable && ev.from == EvSource::Server)
+		if !self.config.fndecls.is_empty()
+			|| self
+				.config
+				.evdecls
+				.iter()
+				.any(|ev| ev.evty == EvType::Reliable && ev.from == EvSource::Server)
```

Comment:
iirc not has a lower precedence than or, so we should wrap the or in parens and then use not. I might be wrong on that though

Reply:
That's what I want!